nexus_transporter/src/transporters/rmf_transporter.cpp

```diff
@@ -575,8 +745,7 @@ class RmfTransporter : public Transporter
       return;
     }
 
-    // Naively just checks if the waypoints exist on the building map.
-    // TODO(ac): use auctioneer for proper bidding
+    // Checks through building map first

```

Comment:
Can we get rid of this altogether now?

Reply:
I was planning on keeping it, as it's a rather cheap lookup as a first check, before going through the bidding process